open-apparel-registry/open-apparel-registry
# Assign facilities in Holland the NL country code

## Overview

Holland is an alternative country name for Netherlands. We also take this opportunity to also make "The Netherlands" an acceptable alternative.

Connects #1263

## Testing Instructions

* Log in as c2@example.com
* Click Contribute and submit [nl-test.csv](https://github.com/open-apparel-registry/open-apparel-registry/files/6111310/nl-test.csv.zip). Note the list ID that is created
* Run `./scripts/manage batch_process -l {id} -a parse`. Verify that it reports success
* Refresh the list page and verify that all 3 facilities have the country "Netherlands"


nl-test.csv
```
"country","name","address"
"Netherlands","NL Facility A","1234 Main St"
"The Netherlands","NL Facility B","323 Market St"
"Holland","NL Facility C","998 Water St"
```
## Checklist
